frontend/src/app/index.js

```diff
@@ -1,48 +0,0 @@
-import 'babel-polyfill/browser';
```

Comment:
Everything that used to happen in app.js now happens in bootstrap.js, which is common across all pages, and the actual app bundle for the page, which gets rendered by running pages/<page>.js:create() and using inject.js:inject() on the result.